README.md

```diff
@@ -81,3 +81,296 @@ Notes on Test Coverage
 ------------------
 
 You can see your [test coverage](https://github.com/makersacademy/course/blob/main/pills/test_coverage.md) when you run your tests.
+------------------
+
+*Thoughts & Notes*
+------------------
+#Friday 26th of november '21
+I create a interactive menu based on the student directory we created a few weeks ago.
+this should implement basic funtions for the user story 1 and 2
+the user can 
+check the menu (list of prices and dishes)
+can order a meal (add to basket several dishes)
+
+here is the diagram for the application I am trying to build : https://wireframe.cc/7hfxO2
+
+based on the following user stories
+```
+As a customer
+So that I can check if I want to order something
+I would like to see a list of dishes with prices
+
+As a customer
+So that I can order the meal I want
+I would like to be able to select some number of several available dishes
+
+As a customer
+So that I can verify that my order is correct
+I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+As a customer
+So that I am reassured that my order will be delivered on time
+I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+```
+
+#Feature test in irb
+works well until I want to check out:
```

Comment:
It is good that you are documenting the user experience and any issues in detail